getting-started-oasgraph.html

```diff
@@ -226,7 +226,7 @@ <h3 class="feature-heading">Start GraphQL server</h3>
             <hr class="hairline-footer">
             <div class="row text-center">
                 <div class="col-md-6">
-                    <p class="regular"> @ Copyright 2018 StrongLoop, an IBM company</p>
+                    <p class="regular"> @ Copyright 2019 StrongLoop, an IBM company</p>
```

Comment:
I was looking up what the "year" should be, then realized that we should use `&copy;` before the `Copyright` word instead of using `@`.  https://www.w3schools.com/html/html_entities.asp.  I know it's not part of your change, it's probably me who introduced the wrong symbol.

The year should be `2018-2019`.